inst/resources/gitbook/css/plugin-bookdown.css

```diff
@@ -90,7 +90,7 @@ div.theorem, div.lemma, div.corollary, div.proposition, div.conjecture {
 span.theorem, span.lemma, span.corollary, span.proposition, span.conjecture {
   font-style: normal;
 }
-div.proof:after {
+div.proof>p:last-child:after {
```

Comment:
This has been addressed by c59e3dcc84058614bf30749ce8c8f26f2c97a6ee.

Reply:
I kind of remember having tried that, and that was not ok in the output... I may have wrongly test it maybe. 🤷‍♂️